db/tables.py

```diff
@@ -44,15 +45,7 @@ def reflect_table(name, schema, engine):
     return Table(name, metadata, schema=schema, autoload_with=engine)
 
 
-def reflect_table_columns(name, schema, engine):
-    t = reflect_table(name, schema, engine)
-    return [
-        {"name": c.name, "type": c.type} for c in t.columns
-    ]
-
-
-def get_all_table_records(name, schema, engine):
-    t = reflect_table(name, schema, engine)
-    sel = select(t)
-    with engine.begin() as conn:
-        return conn.execute(sel).fetchall()
+def get_query(table, engine):
+    with Session(engine) as session:
+        query = session.query(table)
+        return query
```

Comment:
This is just me not thinking enough about what I was doing.

I didn't realize that SQLAlchemy ORM were moving to `Select` objects in 2.0, I'll use that instead. I wanted to use the ORM style instead of core style because it was more intuitive to me, but make no other changes.